src/libmongoc/tests/test-mongoc-retryable-writes.c

```diff
@@ -813,14 +818,12 @@ retryable_writes_original_error_general_command (void *ctx)
 static void
 test_all_spec_tests (TestSuite *suite)
 {
-   install_json_test_suite_with_check (
-      suite,
-      JSON_DIR,
-      "retryable_writes/legacy",
-      test_retryable_writes_cb,
-      test_framework_skip_if_max_wire_version_less_than_6,
-      test_framework_skip_if_no_crypto,
-      test_framework_skip_if_slow);
+   install_json_test_suite_with_check (suite,
+                                       JSON_DIR,
+                                       "retryable_writes/legacy",
+                                       test_retryable_writes_cb,
```

Comment:
```suggestion
                                       test_retryable_writes_cb,
                                       TestSuite_CheckLive,
```

To skip test when environment variable `MONGOC_TEST_SKIP_LIVE=ON` is set.